src/Magnum/Math/Quaternion.h

```diff
@@ -733,6 +740,57 @@ template<class T> Matrix3x3<T> Quaternion<T>::toMatrix() const {
     };
 }
 
+/* Algorithm from:
+   https://github.com/qt/qtbase/blob/84056cdca11c63ddb9fac3d95fcfa952adaa6ea1/src/gui/math3d/qquaternion.cpp#L507 */
```

Comment:
Could this reference the original code Qt is based off, instead? Are there any differences? Because Qt is LGPL and GPL means problems :)